canonical-web-and-design/vanilla-framework
# Remove dependency on vanilla-framework from docs

## Done

- Builds docs custom stylesheet from local latest version of vanilla instead of npm package.
- Moves docs stylesheets to the same folder as vanilla framework so they can all be built and watched together.
- Removes some unnecessary custom styles for docs.
- Removes vanilla-framework from npm dependencies.
- Updates docker and travis configs to make sure deployments work.

Fixes #2721


## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/ or [demo](https://vanilla-framework-canonical-web-and-design-pr-2828.run.demo.haus/)
- Site should look and work the same as before.
- Run `./run test`, tests should pass
